### PR TITLE
[voicerss] Add support for voices

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
This is a working and fully tested alternative to PR #10167

File: bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java [hunk 1]
```diff
@@ -47,9 +49,12 @@
  * @author Jochen Hiller - Initial contribution
  * @author Laurent Garnier - add support for all API languages
  * @author Laurent Garnier - add support for OGG and AAC audio formats
+ * @author cURLy bOi - add support for all API languages (again) and their respective voices
```
Comment (fwolter) [inline bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java:52]: Who's that???

File: bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java [hunk 2]
```diff
@@ -107,7 +112,58 @@
         SUPPORTED_LOCALES.add(Locale.forLanguageTag("zh-tw"));
     }
 
-    private static final Set<String> SUPPORTED_VOICES = Collections.singleton("VoiceRSS");
+    private static final Map<String, Set<String>> SUPPORTED_VOICES = new HashMap<>();
+    static {
+        SUPPORTED_VOICES.put("ar-eg", new HashSet<>(Arrays.asList("Oda")));
```
Comment (fwolter) [inline bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java:117]:
```suggestion
        SUPPORTED_VOICES.put("ar-eg", Set.of("Oda"));
```

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java:52]: It iwas from the guy who started this enhancement (other PR).
Reoved.

File: bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java (hunk 2 above)
Comment (lolodomo) [reply bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImpl.java:117]: Good proposal. Done.

**Review by fwolter — APPROVED** [review]
LGTM

**Comment by EddieParis** [comment]
Hi, can you update the doc as well, because it is really misleading and hiding your great work. By reading doc I understand there is no way to change voice, especially due to this phrase :

>"It actually supports only one voice: "voicerss:default", which is configured to use 44kHz, mono, 16 bit sampling quality."

I suggest 

> use Parameters -> System Services -> Voice to setup the desired language/voice

And again, thanks for all your great work.

**Comment by lolodomo** [comment]
> I suggest
> 
> > use Parameters -> System Services -> Voice to setup the desired language/voice
> 
> And again, thanks for all your great work.

You are welcome to submit a PR to update the documentation. 
The file is here: https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.voice.voicerss/README.md

**Comment by EddieParis** [comment]
Great Thanks, I'll have a look !

**Pull request merged**